.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,10 +4,10 @@ the requirements below.
 -->
 
 ##### Checklist
-<!-- Remove items that are not applicable. For completed items, change [ ] to [x]. -->
+<!-- Remove items that are not applicable. For completed items, change [ ] to [x]. Leave unchecked if not required -->
```

Comment:
Surely 'items that are not applicable' and checks that are 'not required' are the same thing? And I don't think leaving a box unchecked should be something encouraged, as it's ambiguous whether it's something the person whose submitted the PR still needs to do, or it's not applicable to the PR.

Reply:
I've seen the of case of things being checked in the "not required" case but yes I should adjust that. I think leaving it in and leaving it unchecked is best as it also helps the reviewer think about whether any doc update might be useful if it hasn't been done

Reply:
> I've seen the of case of things being checked in the "not required" case but yes I should adjust that.

Sorry?

> I think leaving it in and leaving it unchecked is best as it also helps the reviewer think about whether any doc update might be useful if it hasn't been done 

hmm... okay ... I'd argue that's not the function of a checklist, but it does still serve a purpose, so fair enough :-)

Reply:
> Sorry?

I'm prestty certain I have seen instances where people have decided "docs are not required" and therefore check the "docs updated" box (presumably to indicate they've thought about it) so I wanted to avoid that :-)

Reply:
Have now removed the initial phrase and left my new one in